rosidl_adapter/rosidl_adapter/cli.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Open Source Robotics Foundation, Inc.
+# Copyright 2018-2021 Open Source Robotics Foundation, Inc.
```

Comment:
No need to update the copyright year. I'm pretty sure we don't usually bump it.

Reply:
I've seen both, but I don't mind strongly. Reverted in https://github.com/ros2/rosidl/pull/576/commits/74e0044bdabae5c6f25ce76c6f591423a26ef4ee.